lib/generators/templates/fuel.rb

```diff
@@ -22,6 +22,10 @@
   # Change number of posts to show per page on blog
   config.paginates_per = 5
 
+  config.authors = ['John Doe', 'Jane Smith']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.